MatrixKit/Assets/MatrixKitAssets.bundle/en.lproj/MatrixKit.strings

```diff
@@ -270,6 +270,7 @@
 "network_error_not_reachable" = "Please check your network connectivity";
 "user_id_placeholder" = "ex: @bob:homeserver";
 "ssl_homeserver_url" = "Home Server URL: %@";
+"receipt_status_read" = "Read: ";
```

Comment:
Please move this key-value at the application level in Vector.strings
You may group it with `read_receipts_list` key.